ENIGMAsystem/SHELL/Audio_Systems/DirectSound/DSsystem.cpp

```diff
@@ -70,7 +70,7 @@ void eos_callback(void* soundID, unsigned src) {
 }
 
 int audiosystem_initialize() {
-  if (get_window_handle() == NULL) {
+  if (enigma::hWnd == NULL || enigma::hInstance == NULL) {
```

Comment:
Why favor direct use of hWnd to an API method?

Reply:
Because Robert said window_handle() is meant to be used by extensions and shouldn't be used by the engine at all.. idk you'd have to ask him I'm just going by what he said in a code comment.

Reply:
The real question is, why not?

Reply:
@JoshDreamland why wrap those variables in an API function that's just going to hold the same information?